src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -542,6 +543,11 @@
         }
     }
 
+    .datagrid-foot-description {
```

Comment:
Actually, the reasoning was was that our classes would match standard HTML elements: 
- `.datagrid-head` <-> `thead`
- `.datagrid-body` <-> `tbody`
- `.datagrid-foot` <-> `tfoot`